lib/appExtensions.ts

```diff
@@ -0,0 +1,72 @@
+// Creates a new App Extension on installation of Sample App
+export async function createAppExtension() {
```

Comment:
Is there any reason this function needs to return a Promise? All of the code inside the function can run synchronously 🙂

Reply:
Oh that was from my prior implementations so that should be removed now Thanks!